test/api/apikeys_test.go

```diff
@@ -0,0 +1,159 @@
+package api
+
+import (
+	"encoding/json"
+	"net/http"
+	"net/url"
+	"testing"
+
+	"github.com/SkynetLabs/skynet-accounts/database"
+	"github.com/SkynetLabs/skynet-accounts/skynet"
+	"github.com/SkynetLabs/skynet-accounts/test"
+	"gitlab.com/NebulousLabs/fastrand"
+	"go.sia.tech/siad/modules"
+)
+
+// testAPIKeysFlow validates the creation, listing, and deletion of API keys.
+func testAPIKeysFlow(t *testing.T, at *test.AccountsTester) {
+	name := test.DBNameForTest(t.Name())
+	r, _, err := at.CreateUserPost(name+"@siasky.net", name+"_pass")
+	if err != nil {
+		t.Fatal(err)
+	}
+	at.Cookie = test.ExtractCookie(r)
+
+	aks := make([]database.APIKeyRecord, 0)
+
+	// List all API keys this user has. Expect the list to be empty.
+	r, body, err := at.Get("/user/apikeys", nil)
```

Comment:
f/u at some point we should clean these hardcoded routes up. Ideally we have a single client implementation that we extend like in skyd and just call the helper on that instead of keeping the same `GET` followed by a `json.Unmarshal` around 8 or so times in more than 1 test function.

Reply:
F/U: https://github.com/SkynetLabs/skynet-accounts/issues/128